{{ cookiecutter.folder_name }}/.travis.yml

```diff
@@ -17,18 +27,31 @@ install:
   # Useful for debugging any issues with conda
   - conda info -a
   #Grab all dependencies
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION coverage pip wheel pytest
+  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION pip wheel pytest
```

Comment:
consider using `dev_requirements.txt` here like in `hutch-python`

Reply:
https://github.com/pcdshub/hutch-python/blob/695c6b75e1b8ecda4a32bccb66d3311363fa6532/.travis.yml#L37